repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -6605,6 +6617,7 @@ public void classificationRefreshPropagationV2_new(Map<String, Object> parameter
             }
         }
         LOG.info("classificationRefreshPropagationV2_new: Successfully completed scalable refresh for tag '{}' from source entity {}", classificationTypeName, sourceEntityGuid);
+        return expectedPropagatedVertexIds.size();
```

Comment:
For refresh propagation, it should be sum of tagsToDelete.size() & expectedPropagatedVertexIds.size()